rpc/regen.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/sh
 
-protoc -I. api.proto --go_out=plugins=grpc:walletrpc
+build-protoc-gen-go() {
```

Comment:
FYI, this is failing for me (specifically `bash --posix regen.sh` fails). Not a blocking issue, but might trip up someone in the future.

Reply:
Not sure why bash behaves that way.  I'm testing with OpenBSD's ksh here, and when enabling opt-in support for POSIX compliance (set -o posix) this script doesn't cause any issues.

Reply:
Read up on the POSIX spec.  Functions names matching the regex `[a-zA-Z_][a-zA-Z0-9_]*` must be valid but implementations are allowed to allow other characters as an extension.

I'm not particularly worried about this since it looks like `dash` (the `/bin/sh` on many Linuxes these days) works fine with this.